gloo/docs/installation/setup_kubernetes.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Kubernetes - Prep
+weight: 1
+---
+
+In this document we will review how to prepare different Kubernetes environments before installing Gloo or Gloo
+Enterprise. In general, Gloo and Gloo Enterprise does not require much special in your Kubernetes cluster other than
+in some environments you need to ensure that, at least for installation, the installer has cluster admin rights to be
+able to install the CRDs (Custom Resource Definitions) that need to be installed.
+
+Details for specific Kubernetes distributions:
+
+{{% button href="#minikube" %}}Minikube{{% /button %}}
+{{% button href="#minishift" %}}Minishift{{% /button %}}
+{{% button href="#gke" %}}Google Kubernetes Engine (GKE){{% /button %}}
+{{% button href="#aks" %}}Azure Kubernetes Service (AKS){{% /button %}}
+{{% button href="#eks" %}}Amazon Elastic Container Service for Kubernetes (EKS){{% /button %}}
+
```

Comment:
This looks cool, but wonder if just bullet points would catch the eye better. Maybe someone else can comment on this? @rettori